.github/workflows/test-build-release.yaml

```
@@ -34,6 +35,15 @@ jobs:
         with:
           name: Coverage results
           path: ${{ steps.unity-test-step.outputs.coveragePath }}
+      - name: SonarCloud Scan
+        uses: sonarsource/sonarcloud-github-action@v2.2.0
+        env:
+          SONAR_TOKEN: ${{ secrets.SONAR_TOKEN }}
+        with:
+          args: >
+            -Dsonar.organization=team-zomsa
+            -Dsonar.projectKey=team-zomsa_aplib.net-demo
+            -Dsonar.cs.opencover.reportsPaths=${{ steps.unity-test-step.outputs.coveragePath }}/aplib.net-demo-opencov/EditMode/TestCoverageResults_0000.xml,${{ steps.unity-test-step.outputs.coveragePath }}/aplib.net-demo-opencov/PlayMode/TestCoverageResults_0000.xml,${{ steps.unity-test-step.outputs.coveragePath }}/aplib.net-demo-opencov/EditMode/TestCoverageResults_0001.xml,${{ steps.unity-test-step.outputs.coveragePath }}/aplib.net-demo-opencov/PlayMode/TestCoverageResults_0001.xml
```

If I am not mistaken, we can use wildcards in the reportPaths, meaning we can simplify them:
```suggestion
            -Dsonar.cs.opencover.reportsPaths=${{ steps.unity-test-step.outputs.coveragePath }}/aplib.net-demo-opencov/**/TestCoverageResults_*.xml
```

---

good point